src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/enable-logs-monitored-containers.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Enable logs for instrumented containers
+tags:
+  - Infrastructure
+  - Install the infrastructure agent
+  - Linux installation
+metaDescription: New Relic's infrastructure monitoring agent automatically instruments container runtimes to collect container metrics and metadata.
+freshnessValidatedDate: never
+---
+
+import infrastructureAddNamesLogs from 'images/infrastructure_screenshot-crop_add-names-logs.webp'
+
+import infrastructureAddContainersLogPath from 'images/infrastructure_screenshot-crop_add-containers-log-path.webp'
+
+import logsParsingUi from 'images/infrastructure_screenshot-full_logs-parsing-ui.webp'
+
+Our [infrastructure agent](/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring) can gather logs from your containers. When you enable logs from the infrastructure agent, the agent will automatically assign those logs to the hosts running Docker Compose.
```

Comment:
The Docker Compose mention is not 100% accurate. The containers do not need to be running as part of a Docker Compose setup, they could have been ran  'manually' and the infrastructure-agent would be able to pick the logs up all the same.

Reply:
I just tested with a plain Docker container running manually using 'docker run ...' and the same applies. I need to run `sudo docker run --log-opt tag="{{.Name}}" -d log-generator` in order to get that tag added as attribute.

I will update the docs page accordingly.

Reply:
Hi @harrykimpel , sorry I didn't notice this was specific to Docker Compose. Thanks for clarifying!

Reply:
> Hi @harrykimpel , sorry I didn't notice this was specific to Docker Compose. Thanks for clarifying!

@DavSanchez it was actually a great comment since I was not aware that the same concept also works for manually running Docker containers.